examples/pytorch/graphsaint/README.md

```diff
@@ -0,0 +1,118 @@
+# GraphSAINT
+
+This DGL example implements the sampling method proposed in the paper [GraphSAINT: Graph Sampling Based Inductive Learning Method](https://arxiv.org/abs/1907.04931).
```

Comment:
"implements the sampling method proposed in the paper" -> "implements the paper"